src/modules/CreateChannel/components/InviteUsers/index.tsx

```diff
@@ -21,10 +21,11 @@ import {
   createDefaultUserListQuery,
 } from './utils';
 import { noop } from '../../../../utils/utils';
+import { UserListQuery } from '../../../../types';
 
 export interface InviteUsersProps {
   onCancel?: () => void;
-  userListQuery?(): ApplicationUserListQuery;
+  userListQuery?(): UserListQuery;
```

Comment:
This is for inviting all application users to the new channel so it should be ApplicationUserListQuery no?

Reply:
The userListQuery of InviteUsers is received from CreateChannelProvider. 
<img width="700" alt="image" src="https://github.com/sendbird/sendbird-uikit-react/assets/26326015/a2d07f6f-6342-4deb-98c4-839d65ddf78a">

So, I've changed type to the same type as the one declared internally, which is UserListQuery.
<img width="392" alt="image" src="https://github.com/sendbird/sendbird-uikit-react/assets/26326015/209b5cca-2633-43f1-ac9d-603e250b4b9a">